units/getty@.service.m4

```diff
@@ -46,10 +46,6 @@ KillMode=process
 IgnoreSIGPIPE=no
 SendSIGHUP=yes
 
-# Unset locale for the console getty since the console has problems
-# displaying some internationalized messages.
-Environment=LANG= LANGUAGE= LC_CTYPE= LC_NUMERIC= LC_TIME= LC_COLLATE= LC_MONETARY= LC_MESSAGES= LC_PAPER= LC_NAME= LC_ADDRESS= LC_TELEPHONE= LC_MEASUREMENT= LC_IDENTIFICATION=
```

Comment:
Right. We go to some effort to set the font and utf8 mode on the console, so the removed comment most likely hasn't been true for a long time.